ha/core/system_health/bootstrap/json_kv_convertor.py

```diff
@@ -43,85 +52,158 @@ def __init__(self):
+        """
         if re.search(self._filter_list[0].replace(':',''), key):
             self._uid = val
         elif re.search(self._filter_list[1].replace(':',''), key):
             self._last_modified = val
         elif re.search(self._filter_list[2].replace(':',''), key):
             self._status = val
+            if self._status.upper() in ["NA", "UNKNOWN", "N/A"]:
+                return
             # cleanup the key
-            self._key = key.replace('sites.racks.cortx_nodes.','').replace('.health.status','')
-
-            # create health event
-            self._healthEvent.create_health_event(self._key, self._uid, self._last_modified, self._status, self._conf_index, self._conf_store)
+            # Previous key is node.compute....health.status or node.storage....helath.status
+            # Remove node. and .health.status and form a new key as compute... or storage...
+            self._key = key.replace('node.', '').replace('.health.status','')
+            self._health_event.create_health_event(self._key, self._uid, self._last_modified, self._status, self._conf_index, self._conf_store)
             self._uid = self._last_modified = self._status = self._key = None
 
-
-    def _update_kv(self, k , v):
-
-        kv = f"{k}:{v}\n"
-
-        for pattern in self._filter_list:
-            if re.search(pattern, kv):
-                if pattern == "uid:":
-                    if re.search("health.specifics", kv) or re.search("sites.uid:", kv) or re.search("sites.racks.uid:", kv) or  re.search("sites.racks.cortx_nodes.uid:", kv) :
-                        break
-                elif  re.search("sites.racks.cortx_nodes.last_updated:", kv):
-                    break
-
-                self._update_health(k,v)
-
-
-    def _walk(self, d):
-
-        for k,v in d.items():
-            if isinstance(v, dict):
-                self._path.append(k)
-                self._walk(v)
-                self._path.pop()
-
-            elif isinstance(v, list):
-                self._path.append(k)
-                if not v:
-                    key = (".".join(self._path))
-                    val = v
-                    self._update_kv(key, val)
-                    self._path.pop()
-
-                else:
-                    if isinstance(v[0], dict):
-                        for v_int in v:
-                            self._walk(v_int)
-                    # End value in the KV  is a list
-                    else:
-                        key = (".".join(self._path))
-                        val = v
-                        self._update_kv(key, val)
-                    self._path.pop()
-
-            else:
-                # conditions handled isinstance(v, str) or isinstance(v, int) etc
-                self._path.append(k)
-                key = (".".join(self._path))
-                val = v
-                self._update_kv(key, val)
-                self._path.pop()
+    def _get_required_compute_kv(self, key=None):
+        """
```

Comment:
Pls ensure that this creates keys for all components and subcomponents inside compute

Reply:
Yes. I have checked and mostly I verified it also. If you can see anything, we can connect again.